### PR TITLE
Enable CD Permissions on google-oauth-plugin

### PR DESCRIPTION
**Pull request by julieheard**
Hi, 

I would like to enable automated releases in this repo to make it easier to ensure merged PRs make it into a new release in a timely fashion. I am following: https://www.jenkins.io/doc/developer/publishing/releasing-cd/

I also changed the incrementals version in this repo to be the newest version [here](https://github.com/jenkinsci/incrementals-tools).

If you are happy to enable CD in this plugin, please approve this PR. As part of the process of enabling CD, I have also made [ a PR on jenkins-infra/repository-permissions-updater](https://github.com/jenkins-infra/repository-permissions-updater/pull/3567). Please could you also go to this linked PR and approve that as well.

Many thanks 😄 

```[tasklist]
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -66,8 +66,8 @@
 
   <!-- Bring some sanity to version numbering... -->
   <properties>
-    <revision>1.0.8</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <revision>0</revision>
```
Comment (NotMyFault) [inline pom.xml]:
```suggestion
    <revision>1</revision>
```
Using any number lower than 1, starts cutting releases considered older than 1.0.8

File: pom.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
       Lots of boilerplate from: https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins
   -->
   <artifactId>google-oauth-plugin</artifactId>
-  <version>1.1.2-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```
Comment (NotMyFault) [inline pom.xml]:
```suggestion
  <version>${revision}.${changelist}</version>
```

**Review by NotMyFault — CHANGES_REQUESTED** [review]
The `<scm>` tag pattern should match `<tag>${scmTag}</tag>`

**Comment by julieheard** [comment]
Thank you for your help on this PR @NotMyFault, I really appreciate your time 🙂

**Review by NotMyFault — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -61,13 +61,13 @@
...
     <url>https://github.com/${gitHubRepo}</url>
-    <tag>google-oauth-plugin-1.0.9</tag>
+    <tag>${scmTag}</tag>
   </scm>
 
   <!-- Bring some sanity to version numbering... -->
   <properties>
-    <revision>1.0.8</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <revision>1</revision>
```
Comment (rsandell) [inline pom.xml:69]:
```suggestion
    <revision>2</revision>
```

**Review by rsandell — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (rsandell) [reply pom.xml:69]: I changed my mind, it is ok, since we never hit >= 1.1

**Review by rsandell — APPROVED** [review]

**Comment by rsandell** [comment]
But it looks like you edited an old pom file? version  1.1.1 is the latest release.

**Pull request merged**